src/promptflow-tools/promptflow/tools/template_rendering.py

```diff
@@ -1,9 +1,31 @@
 # Avoid circular dependencies: Use import 'from promptflow._internal' instead of 'from promptflow'
 # since the code here is in promptflow namespace as well
 from promptflow._internal import tool
-from promptflow.tools.common import render_jinja_template
+from promptflow.tools.common import (
```

Comment:
add uts for prompt tool

Reply:
added, please check test_template rendering